tests/unit/cli/test_main.py

```diff
@@ -336,3 +338,78 @@ def test_project_property_works(
         project1, project2 = n.project, n.project
         assert project1 == project2
         assert from_config_mock.call_count == 1
+
```

Comment:
What about avoiding mocks and simply use a tmpdir? Mocking `os.path.exists` makes our test fragile because it would still pass even if the path passed to `os.path.exists()` was wrong.